packages/libclang/runtime.json

```diff
@@ -0,0 +1,59 @@
+{
+  "runtimes": {
```

Comment:
LLVM supports 12 platforms, of those, .NET Core supports 11 (no `PowerPC` support).